nixos/lib/default.nix

```diff
@@ -0,0 +1,46 @@
+let
+  # The warning is in a top-level let binding so it is only printed once.
+  inherit (import ../../lib) warn;
+  experimentalWarning = warn "lib.nixos.evalModules is experimental and subject to change. See nixos/lib/default.nix" null;
+in
+{ lib ? import ../../lib, ... }:
+let
+
+  /*
+    Invoke NixOS. Unlike traditional NixOS, this does not include all modules.
+    Any such modules have to be explicitly added via the `modules` parameter,
+    or imported using `imports` in a module.
+
+    A minimal module list improves NixOS evaluation performance and allows
+    modules to be independently usable, supporting new use cases.
+
+    Parameters:
+
+      modules:        A list of modules that constitute the configuration.
+
+      specialArgs:    An attribute set of module arguments. Unlike
+                      `config.modules.args`, these are available for use in
+                      `imports`.
+
+    Return:
+
+      An attribute set containing `config.system.build.toplevel` among other
+      attributes. See `lib.evalModules`.
```

Comment:
You should specify where one can find this function

Reply:
`evalModules` isn't in the Nixpkgs manual yet (nor NixOS manual, which would be "wrong").
Would love to fix that, but that's for another time.

Changing it to `lib.evalModules` in the Nixpkgs library.